File: Examples/Ruby/DocumentProperties/get_document_property.rb
require 'aspose_words_cloud'

class WorkingWithDocumentProperties

  include AsposeWordsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @words_api = WordsApi.new
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def get_document_property
    file_name = "SampleWordDocument.docx"
    upload_file(file_name)

    property_name = "Author"
    response = @words_api.get_document_property(file_name, property_name)
  end

end

workingWithDocumentProperties = WorkingWithDocumentProperties.new()
puts workingWithDocumentProperties.get_document_property

File: Examples/PHP/Document Properties/GetDocumentProperty.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Words\WordsApi;
use Aspose\Words\AsposeApp;

class DocumentProperties {

	public $wordsApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->wordsApi = new WordsApi();
	}	

	public function getDocumentProperty() {   
		// Upload file to Aspose Cloud Storage
		$fileName = "SampleWordDocument.docx";
		Utils::uploadFile($fileName);

        $result = $this->wordsApi->GetDocumentProperty($fileName, $propertyName="Author", $storage = null, $folder = null);        
        print_r($result);
	}
}

$documentProperties = new DocumentProperties();
$documentProperties->getDocumentProperty();

?>
//ExEnd:
